back/app/controllers/web_api/v1/stats_initiatives_controller.rb

```diff
@@ -2,8 +2,7 @@
 
 class WebApi::V1::StatsInitiativesController < WebApi::V1::StatsController
   def initiatives_count
-    initiatives = StatInitiativePolicy::Scope.new(current_user, Initiative.published).resolve
-      .where(published_at: @start_at..@end_at)
+    initiatives = policy_scope(Initiative.published).where(published_at: @start_at..@end_at)
```

Comment:
I don't think we should need initiatives anymore. I wonder why this wasn't deleted during the cleanup 🤔

Reply:
If this is truly dead code, I can try tackling a small part of it now. I’ve noticed a few places where this could really simplify the policy code.